src/helpers/loadDocument.js

```diff
@@ -71,10 +71,14 @@ const extractXodOptions = options => {
   const xodOptions = {};
 
   if (options.decryptOptions) {
-    xodOptions.decrypt = window.CoreControls.Encryption.decrypt;
+    xodOptions.decrypt = options.decrypt ? options.decrypt : window.CoreControls.Encryption.decrypt;
     xodOptions.decryptOptions = options.decryptOptions;
   }
 
+  if (options.decrypt) {
+    xodOptions.decrypt = options.decrypt;
+  }
```

Comment:
This may be overkill. Im just not sure if custom decrypt also requires decryptOptions.

Reply:
I am not sure actually. @mparizeau-pdftron can you take a look when you have time? Thanks.

Reply:
I think in theory you might have it by itself. If we have this if statement I think we could revert the previous code to `xodOptions.decrypt = window.CoreControls.Encryption.decrypt;` because if there is a custom decrypt function defined then it will override it here.